README.md

```diff
@@ -115,7 +115,7 @@ java -jar app.jar --spring.main.web-application-type=none --price-import --suppl
 ```
 # Import the reporting data for the supplied dates from S3
 
-java -jar app.jar --spring.main.web-application-type=none --report-import --from=YYYY-MM-DD --to=YYYY-MM-DD
+java -Xmx1024m -jar app.jar --spring.main.web-application-type=none --report-import --from=YYYY-MM-DD --to=YYYY-MM-DD
```

Comment:
Small tweak to the docs around the manual running of this.  The need to actually do this manually in environments other locally should be reduced with the introduction of the scheduled task.